webapp/app/[locale]/_components/navbar/_components/getStarted.tsx

```diff
@@ -46,17 +46,12 @@ export const GetStarted = function () {
           }
           `}
         >
-          <GetStartedBackground className="h-20 w-full" />
-          <div className="absolute left-4 top-4 z-20 flex items-center gap-x-1 text-sm">
+          <GetStartedBackground className="h-13 w-full" />
```

Comment:
Actually, the SVG height itself matches the container one, so defining the height isn't needed at all. Removed here [adfe952](https://github.com/hemilabs/ui-monorepo/pull/619/commits/adfe952c1a5e592a8510182e40687b23e26b3c17). Good suggestion otherwise, thanks for pointing it out!